src/qilib/utils/serialization.py

```diff
@@ -18,61 +18,184 @@
 OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 """
 
+from json import JSONEncoder, JSONDecoder
+from typing import Any, Dict, Callable, ClassVar
+from qilib.data_set import MongoDataSetIO
 import numpy as np
-import base64
-import pickle
-import serialize as serialize_module
 
+TransformFunction = Callable[[Any], Any]
```

Comment:
Shoud [Any] be Any?

Add a docstring

Reply:
Yes, `[Any]` is correct. I don't think type aliases should be documented, but will address this.